cookbook/security/api_key_authentication.rst

```diff
@@ -28,7 +28,9 @@ allows you to implement such a scheme really easily.
 
 Your exact situation may differ, but in this example, a token is read
 from an ``apikey`` query parameter, the proper username is loaded from that
-value and then a User object is created::
+value and then a User object is created:
+
+.. code-block:: php
```

Comment:
Ah, ok. Reason I changed it everywhere is that the `code-block` notation was being used in the `Handling Authentication Failure` section at line 294 in this document. Should that be changed to a double colon instead then?